test/js/bun/shell/bunshell.test.ts

```diff
@@ -833,6 +833,11 @@ ${temp_dir}`
       .stdout("complex > command; $(execute)\n")
       .runAsTest("complex_mixed_special_chars");
   });
+
+  test("spaces in path and args", async () => {
+    const fields = "name scripts";
+    expect(async () => { await $`npm pkg get ${fields}` }).not.toThrow();
```

Comment:
note that this test case does not reproduce the original issue, since `bun install` does not use `.cmd` files.

can you create a cmd script using `tempDirWithFiles` and running the a script with that directory in `$PATH`